soc/nordic/sysbuild.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if(SB_CONFIG_VPR_LAUNCHER)
+  set(launcher_core "cpuapp")
+  string(REPLACE "/" ";" launcher_quals ${BOARD_QUALIFIERS})
+  list(GET launcher_quals 1 launcher_soc)
+  string(CONCAT launcher_board ${BOARD} "/" ${launcher_soc} "/" ${launcher_core})
+
+  set(image vpr_launcher)
+  ExternalZephyrProject_Add(
+    APPLICATION ${image}
+    SOURCE_DIR ${ZEPHYR_BASE}/samples/basic/minimal
+    BOARD ${launcher_board}
+  )
+
+  if(SB_CONFIG_VPR_LAUNCHER_XIP)
+    sysbuild_cache_set(VAR ${image}_SNIPPET APPEND REMOVE_DUPLICATES nordic-ppr-xip)
+  else()
+    sysbuild_cache_set(VAR ${image}_SNIPPET APPEND REMOVE_DUPLICATES nordic-ppr)
+  endif()
```

Comment:
The problem with this is it is sticky and does not change. Configure an application using sysbuild and enable the Kconfig for this, then run `ninja sysbuild_menuconfig` and disable this Kconfig and run ninja again, note that the snippet has not changed:
```
-- Board: nrf54h20dk, qualifiers: nrf54h20/cpuapp
-- Snippet(s): nordic-ppr-xip
...
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: nrf54h20dk, qualifiers: nrf54h20/cpuapp
-- Snippet(s): nordic-ppr-xip
```